homeassistant/components/media_player/squeezebox.py

```diff
@@ -201,11 +201,18 @@ def media_duration(self):
     def media_image_url(self):
         """ Image url of current playing media. """
         if 'artwork_url' in self._status:
-            return self._status['artwork_url']
-        return ('http://{server}:{port}/music/current/cover.jpg?'
-                'player={player}').format(server=self._lms.host,
-                                          port=self._lms.http_port,
-                                          player=self._id)
+            media_url = self._status['artwork_url']
+        else:
+            media_url = ('/music/current/cover.jpg?'
+                         'player={player}&'
+                         'nocache={nocache}').format(
+                             player=self._id,
+                             nocache=hash(self.media_title))
```

Comment:
The latest stable version of the server is 7.7.5, offered by mysqueezebox.com/download.
The dev version is 7.9.

The relative url-thing is because of functionality added post-7.7.5: https://github.com/Logitech/slimserver/commit/7478a0bf3ca76d59e3c9c36d2e538b13eedc4323)

This might explain why you didn't see this issue, @persandstrom (in case you were running the stable version?).
The change should be backwards compatible however, since it still will handle absolute artwork url:s correctly.